src/google/Timeline.js

```diff
@@ -28,7 +28,8 @@
     Timeline.prototype.publish("timelineGroupByRowLabel", true, "boolean", "If set to false, creates one row for every dataTable entry. The default is to collect bars with the same row label into one row.",null,{tags:["Basic"]});
     Timeline.prototype.publish("timelineShowBarLabels", true, "boolean", "If set to false, omits bar labels. The default is to show them.", null,{tags:["Basic"]});
     Timeline.prototype.publish("timelineShowRowLabels", true, "boolean", "If set to false, omits row labels. The default is to show them.",null,{tags:["Basic"]});
-    Timeline.prototype.publish("timelineSingleColor", null, "string", "Colors all bars the same. Specified as a hex value (e.g., '#8d8').",null,{tags:["Basic"]});
+    Timeline.prototype.publish("timelineSingleColor", null, "html-color", "Colors all bars the same. Specified as a hex value (e.g., '#8d8').",null,{tags:["Basic"]});
+    Timeline.prototype.publish("timeFormat", "%Y-%m-%d", "string", "Time format of the data.",null,{tags:["Basic"]});
```

Comment:
rename to "timePattern" - I am trying to keep "format" for text we format before displaying and "pattern" as a pattern we use to "deformat" text coming from the server.